extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js

```diff
@@ -1367,7 +1297,13 @@ export class AmpLightboxGallery extends AMP.BaseElement {
         toggle(this.element, true);
       }
 
+
       setStyles(this.element, {
+        // These are needed to stack correctly given that we give the element
```

Comment:
So this is less than ideal right now. With `auto-lightbox` many fixed headers are indexed higher, which causes rendering issues. Not an issue for now but we need to keep it in mind, not sure if the `FixedLayer` hacks actually break anything.

Also moving this here has the side effect of these not being appended to the z-index file.

I'd say just create a fake (obviously namespaced) selector in the CSS for this extension, just for documenting, and regenerate `css/Z_INDEX.md` (`gulp get-zindex`).

Reply:
(Also, personal TODO: there's no reason why these shouldn't be able to be parsed from the get z index generation, it should be trivial to do.)

Reply:
Moved most of the styles into CSS and refactored the controls fade out logic.